nupic/research/frameworks/pytorch/hooks/__init__.py

```diff
@@ -0,0 +1,24 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
```

Comment:
You should update the software licenses (in all files) to say 2021 instead of 2020.

Reply:
Good catch. Updated.